README.md

```diff
@@ -1,70 +1,57 @@
 
-## Building the Mbed OS application
+## Application functionality
 
-#### Import the example
+The `main()` function queries the PSA version and toggles the state of a digital output connected to an LED on the board.
 
-1. From the command line, clone the repository containing the example:
-    ```
-    $ git clone https://github.com/ARMmbed/mbed-os-example-psa
-    ```
+## Building and running
 
-1. Change the current directory to where the project was imported.
+1. Connect a USB cable between the USB port on the target and the host computer.
+1. Run the following command to build the example project, program the microcontroller flash memory, and open a serial terminal:
 
-1. Update `mbed-os` sources by running `mbed deploy`.
+   ```
+   $ mbed compile -m <TARGET> -t <TOOLCHAIN> --flash --sterm --baud 115200
+   ```
 
-#### Compile the example
+Your PC may take a few minutes to compile your code.
 
-Invoke `mbed compile`, and specify the name of your platform and toolchain (`GCC_ARM`, `ARM`). For example, for the ARM_MUSCA_S1 platform and GCC_ARM toolchain:
+The binary is located at `./BUILD/<TARGET>/<TOOLCHAIN>/mbed-os-example-psa.bin`.
+
+## Expected output
+
+The LED on your target turns on and off every 1000 milliseconds. For TFM targets, for example ARM_MUSCA_S1, the serial terminal shows an output similar to:
 
 ```
-mbed compile -m ARM_MUSCA_S1 -t GCC_ARM
-```
-Once its compiled, you should see the following result:
-```
-| Module           |         .text |       .data |          .bss |
-|------------------|---------------|-------------|---------------|
-| [fill]           |       46(+46) |       0(+0) |       37(+37) |
-| [lib]/c.a        |   4792(+4792) | 2108(+2108) |       89(+89) |
-| [lib]/gcc.a      |     764(+764) |       0(+0) |         0(+0) |
-| [lib]/misc       |     180(+180) |       4(+4) |       28(+28) |
-| [misc]           |         0(+0) |       0(+0) |         0(+0) |
-| main.o           |     100(+100) |       0(+0) |         0(+0) |
-| mbed-os/cmsis    |   6716(+6716) |   240(+240) | 14929(+14929) |
-| mbed-os/drivers  |     480(+480) |       0(+0) |         0(+0) |
-| mbed-os/hal      |   2670(+2670) |       8(+8) |     232(+232) |
-| mbed-os/platform |   4090(+4090) |   260(+260) |     268(+268) |
-| mbed-os/rtos     |       28(+28) |       0(+0) |         0(+0) |
-| mbed-os/targets  |   3242(+3242) |     36(+36) |       77(+77) |
-| Subtotals        | 23108(+23108) | 2656(+2656) | 15660(+15660) |
-Total Static RAM memory (data + bss): 18316(+18316) bytes
-Total Flash memory (text + data): 25764(+25764) bytes
-
-Image: ./BUILD/ARM_MUSCA_S1/GCC_ARM/mbed-os-example-psa.bin
+--- Terminal on /dev/tty.usbmodem146102 - 115200,8,N,1 ---
+[INF] Starting bootloader
+[INF] Swap type: none
+[INF] Swap type: none
+[INF] Bootloader chainload address offset: 0x20000
+[INF] Jumping to the first image slot
+[Sec Thread] Secure image initializing!
+Booting TFM v1.1
+PSA Framework Version : 256
```

Comment:
We should avoid putting output which might change, for example, TF-M and PSA version or say x.x

Reply:
There is an argument for both options. I will update the comment and output.